app/code/Magento/InventoryLowStockNotification/Model/ResourceModel/Product/Lowstock/Collection.php

```diff
@@ -187,7 +189,28 @@ public function useNotifyStockQtyFilter(): Collection
             SourceItemInterface::QUANTITY . ' < ?',
             $notifyStockExpression
         );
-        
+
+        return $this;
+    }
+
+    /**
+     * Remove all disabled sources
+     *
+     * @return Collection
+     */
+    public function filterDisabledStocks(): Collection
```

Comment:
Also, need to add this conditions (via search criteria) to `\Magento\InventoryLowQuantityNotification\Model\OptionSource\SourceCode`
And rename to `ActiveSourceCode`